lib/lhm/printer.rb

```diff
@@ -42,7 +42,7 @@ def write(message)
     end
 
     class Dot < Base
-      def notify(lowest = nil, highest = nil)
+      def notify(_lowest = nil, _highest = nil)
```

Comment:
looking at this block, it doesn't look like lowest and highest are used.  But possibly necessary because the extend the class, which is why I assume they were set to nil in the first place.